wiki/Featured_Artists/pt-br.md

```diff
@@ -1,17 +1,41 @@
 ---
-no_native_review: true
-outdated: true
-outdated_since: 84df5a5c33bb4256a47c830bc67b6f3bb7636846
+tags:
+  - FA
+  - Artistas em destaque
 ---
 
 # Artistas em destaque
 
-**Artistas em destaque** são músicos que trabalharam junto ao osu! para fazer uma seleção de suas próprias músicas disponíveis para uso irrestrito na comunidade. Alguns desses artistas em destaque também comporam músicas específicamente para o osu! e lançaram elas como exclusivas do osu!
+**Artistas em destaque** são musicas de artistas que trabalharam com o osu! para fazer uma seleção de suas faixas ficarem disponíveis para uso em [beatmaps](/wiki/Beatmap) e imagem de fundo em músicas de transmissões ao vivo da [Copa do mundo osu!](/wiki/Tournaments#osu!-world-cup).
```

Comment:
Any reason for this particular change? The old wording is more accurate

Reply:
not really, i forgot the old one and translated without verifying it

i'm going to pay attention about that